eZ/Publish/Core/MVC/Symfony/Controller/Content/ViewController.php

```diff
@@ -102,41 +99,9 @@ protected function buildResponse($etag = null, DateTime $lastModified = null)
      *
      * @deprecated Since 6.0.0. Viewing locations is now done with ViewContent.
      */
-    public function viewLocation($locationId, $viewType, $layout = false, array $params = array())
+    public function viewLocation(View $view)
```

Comment:
I'm afraid those won't take typehinting, absolutely.

The current trend is to restore the existing method's code, and add a `view` method. We would end up with `ez_content:view`, which does make sense, and looks good. This one would be the one one, and the old ones would be marked as deprecated.

Opinion ?